hr_timesheet_sheet_period/views/hr_timesheet_sheet_view.xml

```diff
@@ -19,8 +19,8 @@
         </field>
     </record>
 
-    <record id="view_hr_timesheet_sheet_filter" model="ir.ui.view">
-        <field name="name">hr_timesheet.sheet.filter</field>
+    <record id="view_hr_timesheet_sheet_filter_current_period" model="ir.ui.view">
```

Comment:
What's the point in renaming?

Reply:
When there are issues with a view it is easier to identify the view when the name has a clue in it. I can leave it as it was if you think this is not appropriate.

Reply:
I don't think it's appropriate since the ids are scoped to the modules anyways, yet it's a non-blocking opinion. I'll leave it to other maintainers :)

Reply:
For me it is not a must thing, I just left it to what it was